test/views/pages/post/static_post_page_test.dart

```diff
@@ -0,0 +1,196 @@
+          matching: find.byKey(UserBarWidget.displayNameTextKey),
+        ),
+      );
+      expect(
+        postUserBarDisplayNameTextWidget is Text &&
+            postUserBarDisplayNameTextWidget.data == post.ownerDisplayName,
+        true,
+      );
+    });
+
+    testWidgets("Check non-comment widgets are displayed", (tester) async {
+      await tester.pumpWidget(emptyPostPageWidget);
+      await tester.pumpAndSettle();
+
+      // Check if the post is displayed
+      final completePost = find.byType(PostPage);
+      expect(completePost, findsOneWidget);
+
+      //Check that the post distance is displayed
+      final postDistance = find.byKey(PostPage.postDistanceKey);
+      expect(postDistance, findsOneWidget);
+
+      //Check that the complete post widget is displayed
+      final completePostWidget = find.byKey(PostPage.completePostWidgetKey);
+      expect(completePostWidget, findsOneWidget);
+
+      //Check that the comment list widget is displayed
+      final commentListWidget = find.byKey(PostPage.commentListWidgetKey);
+      expect(commentListWidget, findsOneWidget);
+
+      //Check that the bottom bar add comment widget is displayed
+      final bottomBarAddComment = find.byKey(PostPage.bottomBarAddCommentKey);
+      expect(bottomBarAddComment, findsOneWidget);
+
+      //Check that the comment user avatar is displayed
+      final commentUserAvatar =
+          find.byKey(BottomBarAddComment.commentUserAvatarKey);
+      expect(commentUserAvatar, findsOneWidget);
+
+      //Check that the add comment text field is displayed
+      final addCommentTextField =
+          find.byKey(BottomBarAddComment.addCommentTextFieldKey);
+      expect(addCommentTextField, findsOneWidget);
+
+      //Check that the post comment button is displayed
+      final postCommentButton =
+          find.byKey(BottomBarAddComment.postCommentButtonKey);
+      expect(postCommentButton, findsOneWidget);
+    });
+
+    testWidgets("Check comments are displayed", (tester) async {
+      await tester.pumpWidget(nonEmptyPostPageWidget);
+      await tester.pumpAndSettle();
+
+      // Check if the post is displayed
+      final completePost = find.byType(PostPage);
+      expect(completePost, findsOneWidget);
+
+      //Check that comment list widget is displayed
+      final commentListWidget = find.byKey(PostPage.commentListWidgetKey);
+      expect(commentListWidget, findsOneWidget);
+
+      //Check that we have the right number of comments
+      final commentList = find.byKey(CommentPostWidget.commentWidgetKey);
+      expect(commentList, findsNWidgets(testComments.length));
+
+      //Check that the comment user widgets are displayed
+      final commentUserAvatar =
+          find.byKey(CommentPostWidget.commentUserWidgetKey);
+      expect(commentUserAvatar, findsNWidgets(testComments.length));
+
+      //Check that the username are displayed
+      final Iterable<Text> displayNameWidgetsWidgets = tester.widgetList<Text>(
```

Comment:
There is a small typo in this variable name

Reply:
_You don't know the story of the WidgetsWidgets? 
It's a legendary tale_

Oops, Fixed in: [23369bb](https://github.com/ProximaEPFL/proxima/pull/143/commits/23369bb7a7e2a74eb1119b85a6c85b3ebee6421b) Thank you !